setup.py

```diff
@@ -7,7 +7,7 @@
 long_description = THIS_DIR.joinpath('README.rst').read_text()
 
 # avoid loading the package before requirements are installed:
-version = SourceFileLoader('version', 'aiohttp_devtools/version.py').load_module()
+version = SourceFileLoader("__version__", "aiohttp_devtools/__init__.py").load_module()
```

Comment:
FYI modern `setuptools` allows you to just have this in `setup.cfg` as something like `version = attr: aiohttp_devtools.__version__`